src/data/list/basic.lean

```diff
@@ -1304,6 +1305,10 @@ end insert_nth
 
 @[simp] lemma map_nil (f : α → β) : map f [] = [] := rfl
 
+theorem map_eq_foldr (f : α → β) (l : list α)
+  : map f l = foldr (λ a bs, f a :: bs) [] l :=
```

Comment:
The style guide says not to start lines with punctuation. 
```suggestion
theorem map_eq_foldr (f : α → β) (l : list α) :
  map f l = foldr (λ a bs, f a :: bs) [] l :=
```

Reply:
And below.

Reply:
Fixed throughout.